cypress/integration/publishingFlows/commentOnAnArticle.spec.js

```diff
@@ -0,0 +1,91 @@
+describe('Comment on an article', () => {
```

Comment:
we also have a `commentOnArticle` suite of tests under `articleFlows` - we should combine the two

Reply:
That makes sense. I realized yesterday when I noticed a test in another PR of mine with the same name failed. 🙃 I'll add the canned responses tests into tour existing test file.